share/spice/twitter/twitter.js

```diff
@@ -35,10 +35,15 @@
                 sourceName: "Twitter",
             },
             normalize: function(item) {
+                var subtitle = ["@" + item.user, ];
```

Comment:
Sorry, habit from Python. It should be ok though. Means that there is just one item in the array.